pkg/apis/scheduling/v1beta1/labels.go

```diff
@@ -42,6 +42,9 @@ const QueueNameAnnotationKey = GroupName + "/queue-name"
 // PodPreemptable is the key of preemptable
 const PodPreemptable = "volcano.sh/preemptable"
 
+// PodPreemptStableTime is the key of preempt-stable-time, value's unit is second
+const PodPreemptStableTime = "volcano.sh/preempt-stable-time"
```

Comment:
l am ok about the changes.
/lgtm

Reply:
why do we introduce this label? can we reuse deletiontimestamp?

Reply:
> why do we introduce this label? can we reuse deletiontimestamp?

this label is used by scheduler to protect these pods not to be preempted in some time, not to make them deleted after some time

Reply:
> l am ok about the label name, but l prefer to use “ns”, “us”, “ms”, “s”, “m”, “h” as time units

good point, I'll add unit support in pr [#2149](https://github.com/volcano-sh/volcano/pull/2149)